dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/utils/FormUtils.java

```diff
@@ -154,6 +154,11 @@ private static CategoryCombo getCategoryCombo( DataSet dataset, Set<Organisation
             {
                 for ( DataElementCategory cat : cats )
                 {
+                    if ( cat.getAccess() != null && !cat.getAccess().isRead() )
```

Comment:
Is it better with?

option.getAccess() == null || !option.getAccess().isRead() )